From 5229430874b5275547babdbef1e322922317456d Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 25 Jul 2019 23:30:27 -0700
Subject: [PATCH 19/19] libfdt: Undefine __wordsize if already defined

glibc 2.30+ defines __wordsize, which is same so its easier to compile
for multiple versions of glibc even ones which does not have this define

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 opensrc/helpers/libfdt/libfdt_env.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/opensrc/helpers/libfdt/libfdt_env.h b/opensrc/helpers/libfdt/libfdt_env.h
index 1c966b8..fc25ca6 100644
--- a/opensrc/helpers/libfdt/libfdt_env.h
+++ b/opensrc/helpers/libfdt/libfdt_env.h
@@ -56,6 +56,10 @@
 #include <stdint.h>
 #include <string.h>
 
+#ifdef __bitwise
+#undef __bitwise
+#endif
+
 #ifdef __CHECKER__
 #define __force __attribute__((force))
 #define __bitwise __attribute__((bitwise))
-- 
2.22.0

